tests/pages/base.py

```diff
@@ -129,3 +133,64 @@ def click_sign_me_up(self):
         def sign_up_successful(self):
             el = self.selenium.find_element(*self._thank_you_locator)
             return el.is_displayed()
+
+    class MozillaNewsletterEmbedForm(PageRegion):
```

Comment:
I started down this route initially but then got swayed on if it created too much complexity, but will give this a revisit. Agree there is a bit too much repetition currently :+1: